test/e2e/network/dns_common.go

```diff
@@ -444,14 +446,20 @@ func createDNSPod(namespace, wheezyProbeCmd, jessieProbeCmd, podHostName, servic
 	return dnsPod
 }
 
-func createProbeCommand(namesToResolve []string, hostEntries []string, ptrLookupIP string, fileNamePrefix, namespace, dnsDomain string) (string, []string) {
+func createProbeCommand(isIpv6 bool, namesToResolve []string, hostEntries []string, ptrLookupIP string, fileNamePrefix, namespace, dnsDomain string) (string, []string) {
```

Comment:
this is a really long param list, can we do this
```
type probeCommandArgs struct {
  ipv6 bool
  namesToResolve []string
  hostEntries []string
  ptrLookupIP string
  fileNamePrefix string
  namespace string
  dnsDomain string
}

func createProbeCommand(a *probeCommandArgs) ...
```

Reply:
this also remains unresolved